setup.py

```diff
@@ -0,0 +1,123 @@
+#
+"""Python setuptools setup."""
+
+import glob
+import os
+import shutil
+from setuptools import setup, Extension, find_namespace_packages
+
+from Cython.Build import cythonize
+
+
+def get_verified_absolute_path(path):
+    """Verify and return absolute path of argument.
+
+    Args:
+        path : Relative/absolute path
+
+    Returns:
+        Absolute path
+    """
+    installed_path = os.path.abspath(path)
+    if not os.path.exists(installed_path):
+        raise RuntimeError("No valid path for requested component exists")
+    return installed_path
+
+
+def get_installation_requirments(file_path):
+    """Parse pip requirements file.
+
+    Args:
+        file_path : path to pip requirements file
+
+    Returns:
+        list of requirement strings
+    """
+    with open(file_path, 'r') as file:
+        requirements_file_content = \
+            [line.strip() for line in file if line.strip() and not line.lstrip().startswith('#')]
+    return requirements_file_content
+
+
+def copy_all_files_in_directory(src, dest, file_ext="*.so"):
+    """Copy files with given extension from source to destination directories.
+
+    Args:
+        src : source directory
+        dest : destination directory
+        file_ext : a regular expression string capturing relevant files
+    """
+    files_to_copy = glob.glob(os.path.join(src, file_ext))
+    if not files_to_copy:
+        raise RuntimeError("No {} files under {}".format(src, file_ext))
+    os.makedirs(os.path.dirname(dest), exist_ok=True)
+    try:
+        for file in files_to_copy:
+            shutil.copy(file, dest)
+            print("{} was copied into {}".format(file, dest))
+    except (shutil.Error, PermissionError) as err:
+        print('Could not copy {}. Error: {}'.format(file, err))
+        raise err
+
+
+# Must be set before calling pip
+for envvar in ['AW_INSTALL_DIR', 'AW_VERSION']:
+    if envvar not in os.environ.keys():
+        raise EnvironmentError(
+            '{} environment variables must be set'.format(envvar))
+
+aw_install_dir = os.environ['AW_INSTALL_DIR']
+aw_version = os.environ['AW_VERSION']
+
+# Get current dir (pyclaragenomics folder is copied into a temp directory created by pip)
+current_dir = os.path.dirname(os.path.realpath(__file__))
+
+# Classifiers for PyPI
+pyaw_classifiers = [
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Science/Research",
+        "Topic :: Scientific/Engineering :: Bio-Informatics",
+        "License :: OSI Approved :: Apache Software License",
+        "Natural Language :: English",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9"
+]
+
+setup(name='atacworks',
+      version=aw_version,
```

Comment:
for now we can hard code the version numbers here. this is a holdover from CGA setup

Reply:
can you add the version number directly to the variable in `setup` call instead of using the other variable? we can add it later if we derive from a variable

Reply:
done.